Source/automap.h

```diff
@@ -79,7 +78,7 @@ void DrawAutomap(const CelOutputBuffer &out);
 /**
  * @brief Marks the given coordinate as within view on the automap.
  */
-void SetAutomapView(int x, int y);
+void SetAutomapView(const Point tile);

```

Comment:
no need for `const` when passing by value generally -- it is up to the function what to do with the copied argument

Reply:
This should be a const reference I think, not a const.

Reply:
Well the general guidance is to pass tiny objects by value

Reply:
Yes, well for me anything > sizeof(int) (in this case, two ints) is not tiny. But not strong feeling about this.

Reply:
On 64bit systems `sizeof(int) * 2` is probably a more reasonable measure as anything smaller takes less than the address for it would.

Reply:
Not sure I understand, on 64 bit systems `sizeof(int) == 64`, which is the same as `sizeof(void*)`. But yeah the common rule is that anything bigger than a pointer should be passed by const references. Or `sizeof(Point) == sizeof(void*) * 2` on any system.

Reply:
You probably mean `sizeof(int) == 8`, but at least on my system (Linux x86_64 GCC) `sizeof(int) == 4`.

Passes:
`static_assert(sizeof(Point) == sizeof(void*));`
Fails:
`static_assert(sizeof(Point) == sizeof(void*) * 2);`

Reply:
> You probably mean `sizeof(int) == 8`, but at least on my system (Linux x86_64 GCC) `sizeof(int) == 4`.

Ah yes sorry, I confused `long` with `int`.